Add @reason-react-native/netinfo

sgny opened the pull request:
`NetInfo` has been removed from RN as of version 0.60 and moved into a community repository.

This package provides bindings to the community package.

glennsl commented:
Thanks. Unfortunately the NPMS indexer seems to be down, and has been for over a month. We'll have to wait until it's up before I can add this.

glennsl commented:
The published readme for this is a bit lacking. You have to publish to npm for any readme changes to get picked up as well. Could you do that?

sgny commented:
@glennsl Sorry for the long delay before responding. This package is being moved into the `reason-react-native` repo and the `npm` package will be `@reason-react-native/netinfo`. I could update the PR with a link when that happens. Would that be alright?

Regarding the readme, I purposely kept the readme for the npm package barebones since it is simpler to update the repo with revised documentation instead of updating the package. I'll revisit the readme to include a bit more detail nevertheless.

glennsl commented:
Yes, please do. I'll keep the PR open.

You can still revise the documentation in the repository even if you publish it, and point from the published readme to the repository for updated docs. But if you don't publish it, and change the API in a later version, then the repository Readme will likely be misleading for earlier versions. That's why Readmes are published as well.

glennsl commented:
@sgny Any update on this?

sgny commented:
My apologies for not following up sooner. We launched an initiative to offer bindings for components extracted from React Native, so I moved the bindings to under the `@reason-react-native` organisation. I have updated the PR accordingly.

I believe `reason-` should be before `reasonql` in ordering, but I would happily revise if mistaken.
